test/functional/feature_reindex.py

```diff
@@ -8,6 +8,7 @@
 - Stop the node and restart it with -reindex. Verify that the node has reindexed up to block 3.
 - Stop the node and restart it with -reindex-chainstate. Verify that the node has reindexed up to block 3.
 - Verify that out-of-order blocks are correctly processed, see LoadExternalBlockFile()
+- Start a second node, generate blocks, then restart with -reindex after setting blk files to read-only
```

Comment:
If you need to retouch, maybe expand this comment to explain why two nodes are needed.

Reply:
I think I added a second node to the test so I could start a clean chain using smaller `blk` files (`-fastprune`) without bugging the existing tests that switch a few options on and off throughout.

Reply:
added comment to top of the test where second node is set up